### DISCUSSION TITLE
Custom Nuclei Template Path

### DISCUSSION THREAD

**Discussion by ghost** [projectdiscovery/nuclei-templates#5562] [Q&A] [answered]
Hi Team 
I Would like ask about private templates .I installed Nuclei .Now I want update with private templates .Where is propel location in Nuclei root. 
![templates](https://user-images.githubusercontent.com/32041737/193905887-8bf91bad-6336-4248-9e54-55fc467c49a0.jpg)

The templates I got from this website 
github.com/Akokonunes/Private-Nuclei-Templates

**Comment by ehsandeep** [comment, marked as answer]
@kolor33 you can place it anywhere inside the directory where default nuclei template were installed, for example, `~/nuclei-templates/`
